v2/internal/pkg/cli/executor.go

```diff
@@ -191,6 +192,13 @@ func NewMirrorCmd(log clog.PluggableLoggerInterface) *cobra.Command {
 			log.Info(emoji.WavingHandSign + " Hello, welcome to oc-mirror")
 			log.Info(emoji.Gear + "  setting up the environment for you...")
 
+			// OCPBUGS-55374 (check current umask)
+			currentUmask := syscall.Umask(0)
+			// 18 represents 22 in octal
+			if currentUmask != 18 {
```

Comment:
What about other umask values other than 022 that still allow oc-mirror to read its own files?

Reply:
The e2e tests failure is an example of that.

Reply:
Other umask other than 0022 will not be considered  i.e. from oc-mirror docs 

```
Prerequisites

You have installed the OpenShift CLI (oc). If you are mirroring image sets in a fully disconnected environment, ensure the following:

-You have installed the oc-mirror plugin on the host that has internet access.
-The host in the disconnected environment has access to the target mirror registry.
-You have set the umask parameter to 0022 on the operating system that uses oc-mirror.
-You have installed the correct binary for the RHEL version that you are using.

```

Reply:
Then this is not a bug and I don't think we need this check.